src/content/en/fundamentals/performance/why-performance-matters/index.md

```diff
@@ -0,0 +1,390 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Thanks to mobile device and network proliferation, more people are
```

Comment:
description should not break - it's only line that's allowed to exceed 80 chars.